templates/geosafe/analysis/notification/email_notification.html

```diff
@@ -1,4 +1,6 @@
 <p>
+    {% blocktrans %}
```

Comment:
The benefit of doing it using template :p.

Don't forget that blocktrans needs: `{% load i18n %}` at the top.

Also you should do the same for email_notification.txt, if it's not already translated.